crowbar_framework/app/helpers/barclamp/suse_manager_client_helper.rb

```diff
@@ -25,7 +25,7 @@ def suse_manager_client_rpm
     end
 
     def suse_manager_client_install
-      "/opt/dell/bin/barclamp_install.rb --rpm suse-manager-client"
+      "knife cookbook upload core -o /opt/dell/chef/cookbooks"
```

Comment:
Ok, let's keep it as it is when we have a button or an API for the lib then we can adapt it later.